### PR TITLE
Disable auto mode for cluster creation

### PR DESCRIPTION
**Pull request by suchita-g**
Signed-off-by: suchita.gatfane <sgatfane@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/rosa.py [hunk 1]
```diff
@@ -58,17 +58,24 @@ def create_cluster(cluster_name, version, region):
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
+    mode = config.ENV_DATA.get("mode", "")
```
Comment (fbalak) [inline ocs_ci/utility/rosa.py:61]: I recommend to call the option rosa_mode or something similar so that it is clear what the mode is used for.

**Comment by suchita-g** [comment]
verified https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/11641/

**Review by fbalak — APPROVED** [review]

File: ocs_ci/utility/rosa.py [hunk 2]
```diff
@@ -58,17 +58,35 @@ def create_cluster(cluster_name, version, region):
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
+    rosa_mode = config.ENV_DATA.get("rosa_mode", "")
```
Comment (ebondare) [inline ocs_ci/utility/rosa.py:61]: I'd suggest making auto the default rosa mode if it's not specified in the config

Comment (ebondare) [reply ocs_ci/utility/rosa.py:61]: OK

**Review by ebondare — APPROVED** [review]
LGTM

**Review by dahorak — APPROVED** [review]

**Pull request merged**